tests/kubevirt_helper.go

```diff
@@ -762,3 +763,19 @@ func DeployVMTemplatesAndValidate() error {
 	_, err = task.DoRetryWithTimeout(waitForCompletedAnnotations, importerPodCompletionTimeout, importerPodRetryInterval)
 	return err
 }
+
+func AddToVMNodeMap(vmName, nodeName string, vmNodeMap *map[string]string) {
+	(*vmNodeMap)[vmName] = nodeName
+}
+
+func ValidateAndUpdateVMNode(vmName, nodeName string, vmNodeMap *map[string]string) error {
```

Comment:
There should not be any need to pass map as a pointer unless the function is returning a brand new map.

Reply:
Fixed.